app/routes/bdb/utils.tsx

```diff
@@ -153,7 +153,7 @@ export const getContactedStatuses = (
   });
   return contacted;
 };
-export const ListNavigation = ({ title }: { title: Node }) => (
```

Comment:
Is `title` ever anything else besides a string? It's wrapped by a `h1` tag.

Reply:
Yup, in the add company page the title is a textfield.

Reply:
well that's dumb but okay 😬